app/controllers/questions_controller.rb

```diff
@@ -70,11 +183,32 @@ def log_time
   end
 
   def next
-    setup = Setup.take
-    week = setup.weeks.find_by_number(params[:week_number])
-    RecommenderSystem::Recommender.setup(current_user.id,week.id)
-    best = RecommenderSystem::HybridRecommender.new.get_best
-    los = LearningObject.find(best[0])
-    redirect_to action: "show", id: los.url_name
```

Comment:
Zatial to ma napevno, ze najviac 10 otazok. To cislo sme dlhsie vyberali a nakoniec sme skoncili na 10. Ak sa tento koncept levelov ujme, mozem to tam doplnit, ale zatial to je asi zbytocne :)